doc/devel/testing.rst

```diff
@@ -130,11 +136,27 @@ images on the figures using two different methods (the tested method and the
 baseline method).  The decorator will arrange for setting up the figures and
 then collect the drawn results and compare them.
 
-It is preferred that new tests use ``style='mpl20'`` as this leads to smaller
-figures and reflects the newer look of default Matplotlib plots. Also, if the
-texts (labels, tick labels, etc) are not really part of what is tested, use
-``remove_text=True`` as this will lead to smaller figures and reduce possible
-issues with font mismatch on different platforms.
+For example, this test compares two different methods to draw the same
+circle: plotting a circle using circle patch (matplotlib.patches.Circle())
```

Comment:
The `Circle` should either be a link (single backticks) or code (double backticks).

Reply:
Okay! You are talking about the Circle() here, right?

Reply:
I just edited it. Please, review. :)